### PR TITLE
documentaion: Update tensorflow tutorial links.  Add a link to hosted TensorBoard

### PR DESCRIPTION
**Pull request by bileschi**
* Motivation for features / changes
Fixes broken links
Adds a way to play with a tensorboard without installing anything.

* Technical description of changes
Just some readme fixes.

* Screenshots of UI changes
capture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ Learning][].
 For in-depth information on the Graph Visualizer, see this tutorial:
 [TensorBoard: Graph Visualization][].
 
-[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/get_started/summaries_and_tensorboard
-[TensorBoard: Graph Visualization]: https://www.tensorflow.org/get_started/graph_viz
+[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/r1/summaries
```
Comment (GalOshri) [inline README.md]: This link 404s for me. Should we point at the newer tutorials https://www.tensorflow.org/tensorboard/get_started?

File: README.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ Learning][].
...
+[TensorBoard: Graph Visualization]: https://www.tensorflow.org/tensorboard/graphs
 
 You may also want to watch
 [this video tutorial][] that walks
```
Comment (GalOshri) [inline README.md]: Are the APIs shown here still the right ones to show our users? I think we should overall emphasize our [new tutorials](https://www.tensorflow.org/tensorboard/get_started) and that they run in Colab.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,13 @@ through setting up and using TensorBoard. There's an associated
 
 [tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/martinwicke/tf-dev-summit-tensorboard-tutorial
 
+You may also be interested in the hosted TensorBoard solution at
```
Comment (GalOshri) [inline README.md]: I'd suggest mentioning the value of this product. I.e. "You can use [TensorBoard.dev](https://tensorboard.dev) to easily host, track, and share your ML experiments for free. For example, ..."

File: README.md (hunk 1 above)
Comment (bileschi) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (bileschi) [reply README.md]: Removed the link to the video tutorial.

File: README.md (hunk 3 above)
Comment (bileschi) [reply README.md]: Added text regarding why you might be interested in tensorboard.dev

File: README.md [hunk 4]
```diff
@@ -5,22 +5,18 @@ TensorFlow runs and graphs.
...
-
-You may also want to watch
-[this video tutorial][] that walks
-through setting up and using TensorBoard. There's an associated
-[tutorial with an end-to-end example of training TensorFlow and using TensorBoard][].
-
-[this video tutorial]: https://www.youtube.com/watch?v=eBbEDRsCmv4
-
-[tutorial with an end-to-end example of training TensorFlow and using TensorBoard]: https://github.com/martinwicke/tf-dev-summit-tensorboard-tutorial
+using TensorBoard, see the tutorial: [TensorBoard: Getting Started][].
```
Comment (GalOshri) [inline README.md]: Markdown link is broken here.

File: README.md [hunk 5]
```diff
@@ -5,22 +5,18 @@ TensorFlow runs and graphs.
...
+Documentation on how to use TensorBoard to work with images, graphs, hyper
+parameters, and more are linked from there, along with tutorial walk-throughs in
+Colab.
+
+You may also be interested in the hosted TensorBoard solution at
+[TensorBoard.dev][].  You can use TensorBoard.dev to easily host, track, and
+share your ML experiments for free.  For example, [this experiment][] shows a
```
Comment (GalOshri) [inline README.md]: Very nit: the rest of the doc seems to have one space after '.'. It doesn't affect the end result but might be nice for consistency.

**Comment by wchargin** [comment]
Closing and reopening to trigger new CI.

File: README.md (hunk 4 above)
Comment (bileschi) [reply README.md]: Good catch. Thanks for checking the links.

File: README.md (hunk 5 above)
Comment (bileschi) [reply README.md]: Done.

**Pull request merged**